README.md

```diff
@@ -1 +1,31 @@
-# AwesomeBooks
\ No newline at end of file
+# AwesomeBooks
+ 
+A simple JavaScript Project that display bok Title and Author. When a user click on the add button, you have the value imputed added at the top of the page and when user clicks on Remove button it automatically remove the value input. Modified the old code since it already match the project requirement.
+
+Added Modules and made use of ES6
+
+
+# Build with 
+ - HTML 
+ - CSS
+ - JavaScript
+
```

Comment:
- [ ] Please add the `Installations` section to your Readme file which explains how to setup and use the project. This would help the users a lot.